src/dyad/modules/dyad.c

```diff
@@ -268,66 +270,68 @@ static void show_help (void)
     DYAD_LOG_STDOUT ("dyad module options and arguments\n");
     DYAD_LOG_STDOUT ("    -h, --help:  Show help.\n");
     DYAD_LOG_STDOUT ("    -d, --debug: Enable debugging log message.\n");
-    DYAD_LOG_STDOUT ("    -m, --mode:  DTL mode. Need an argument.\n"
-                     "                 Either 'FLUX_RPC' (default) or 'UCX'.\n");
-    DYAD_LOG_STDOUT ("    -i, --info_log: Specify the file into which to redirect\n"
-                     "                    info logging. Does nothing if DYAD was not\n"
-                     "                    configured with '-DDYAD_LOGGER=PRINTF'.\n"
-                     "                    Need a filename as an argument.\n");
-    DYAD_LOG_STDOUT ("    -e, --error_log: Specify the file into which to redirect\n"
-                     "                     error logging. Does nothing if DYAD was\n"
-                     "                     not configured with '-DDYAD_LOGGER=PRINTF'\n"
-                     "                     Need a filename as an argument.\n");
+    DYAD_LOG_STDOUT (
+        "    -m, --mode:  DTL mode. Need an argument.\n"
+        "                 Either 'FLUX_RPC' (default) or 'UCX'.\n");
+    DYAD_LOG_STDOUT (
+        "    -i, --info_log: Specify the file into which to redirect\n"
+        "                    info logging. Does nothing if DYAD was not\n"
+        "                    configured with '-DDYAD_LOGGER=PRINTF'.\n"
+        "                    Need a filename as an argument.\n");
+    DYAD_LOG_STDOUT (
+        "    -e, --error_log: Specify the file into which to redirect\n"
+        "                     error logging. Does nothing if DYAD was\n"
+        "                     not configured with '-DDYAD_LOGGER=PRINTF'\n"
+        "                     Need a filename as an argument.\n");
 }
 
 struct opt_parse_out {
-    const char* prod_managed_path;
-    const char* dtl_mode;
+    const char *prod_managed_path;
```

Comment:
Let's refrain from styling changes like these. Unless the pointers are exclusively used by 'C', stick to the convensional C++ style.